pkg/reconciler/common/transformers.go

```diff
@@ -48,6 +48,8 @@ const (
 
 	resultAPIDeployment     = "tekton-results-api"
 	resultWatcherDeployment = "tekton-results-watcher"
+
+	PACImageEnv = "IMAGE_PAC_PIPELINES_AS_CODE"
```

Comment:
Should we consider this whole, or have a specific format? Like Having Triggertemplate in name so that we know this is for triggertemplates, otherwise if we go with the pattern folowed, this will look like it finds a container with name - `pipelines-as-code` in deployment

Reply:
I see how this can be confusing, I'll rename it to `IMAGE_PAC_TRIGGER_TEMPLATES`

Reply:
Fixed in e1baf385c4f2012d01c33ca0783f29ac912f6d24